bundles/ui/org.eclipse.smarthome.ui.basic/README.md

```diff
@@ -0,0 +1,13 @@
+## Eclipse Smarthome BasicUI
```

Comment:
Could you add some more infos on this page? E.g. about the configuration options and maybe some screenshots.